src/java/org/apache/cassandra/service/snapshot/SnapshotOptions.java

```diff
@@ -214,7 +224,7 @@ public SnapshotOptions build()
             if (rateLimiter == null)
                 rateLimiter = DatabaseDescriptor.getSnapshotRateLimiter();
 
-            return new SnapshotOptions(type, tag, ttl, creationTime, skipFlush, ephemeral, entities, rateLimiter,
+            return new SnapshotOptions(type, tag, ttl, creationTime, skipFlush, ephemeral, force, entities, rateLimiter,
```

Comment:
NIT, maybe just pass the builder instead?

```suggestion
            return new SnapshotOptions(this);
```

and the constructor will look like this:

```
    private SnapshotOptions(Builder builder)
    {
        this.type = builder.type;
        this.tag = builder.tag;
        this.ttl = builder.ttl;
        this.creationTime = builder.creationTime;
        this.skipFlush = builder.skipFlush;
        this.ephemeral = builder.ephemeral;
        this.force = builder.force;
        this.entities = builder.entities;
        this.rateLimiter = builder.rateLimiter;
        this.sstableFilter = builder.sstableFilter;
        this.cfs = builder.cfs;
    }
```